packages/react-components/button/src/SocialMediaButtons/index.tsx

```diff
@@ -1,6 +1,6 @@
 import {
-  FacebookBrandIcon,
-  GoogleBrandIcon,
+  FacebookRoundBrandIcon,
```

Comment:
Personally I would avoid using any kind of "style" (round, colored) indicators in component names. I think previous, more generic, names were more appropriate.

Reply:
I think they have different meanings. For instance, there is also FacebookSquareBrandIcon... To be honest I would keep it like this to avoid UI breaking changes but I let the owner of the repo decide of course 😄 cc @aaronbates

Reply:
In future probably we want to have only 1 "correct" version of social media icons. But yeah, makes sense to leave it as it is for now.